shared/js/background/trackers.es6.js

```diff
@@ -309,7 +313,7 @@ function checkABPParsedList (list, url, siteDomain, request) {
  *   * to content scripts to start the process of hiding blocked ads
  *    */
 function tryElementHide (requestData, tab) {
-    if (tab.parentEntity === 'Oath') {
```

Comment:
This is a very small change. I noticed we were already setting the parentEntity on the Tab class because of element hiding. I've changed it to use on the Site class, to avoid getting the parent multiple times unnecessarily. (See also: https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/290/files#diff-50231d3e2a7216164812f45897e7ec77L42)

cc @dharb